src/main/java/marquez/api/ListNamespacesResponse.java

```diff
@@ -20,7 +20,7 @@ public ListNamespacesResponse(@JsonProperty("namespaces") List<Namespace> namesp
   public List<marquez.api.Namespace> getNamespaces() {
     return namespaces
         .stream()
-        .map(namespaceMapper::map)
+        .map(namespaceMapper::mapIfPresent)
         .filter(Optional::isPresent)
```

Comment:
I only updated references to `NamespaceMapper.map()` and didn't look too much into its usage. But, yeah, this could be cleaned up. I added comments to address them later (see [abaf94e](https://github.com/MarquezProject/marquez/pull/155/commits/abaf94e3cddd9963e517465ef7ec69ce6dc4e83e)).